rpc/src/lib.rs

```diff
@@ -47,7 +47,7 @@ pub(crate) fn make_json_stream_response<T: futures::Stream<Item=Result<String, s
         .header(hyper::header::ACCESS_CONTROL_ALLOW_HEADERS, "Content-Type")
         .header(hyper::header::ACCESS_CONTROL_ALLOW_HEADERS, "content-type")
         .header(hyper::header::ACCESS_CONTROL_ALLOW_METHODS, "GET, POST, OPTIONS, PUT")
-        .header(hyper::header::TRANSFER_ENCODING, "chunked")
+        // .header(hyper::header::TRANSFER_ENCODING, "chunked")
```

Comment:
@adonagy 
do we need it or not?

Reply:
We don't. It is the default configuration. No point setting it explicitly.

Reply:
But still, no idea why I left there a same, but commented line